public/index.js

```diff
@@ -1,23 +1,28 @@
+'use strict';
+
 const $urlInput = $('.url-input');
 const $displayUrls = $('.display-urls');
```

Comment:
We already had this variable for the container.  Since you made a new one, can you delete this one and make sure we're not calling it anywhere?  Thanks!